src/sql/src/session/vars/definitions.rs

```diff
@@ -2073,7 +2073,7 @@ feature_flags!(
     {
         name: enable_variadic_left_join_lowering,
         desc: "Enable joint HIR ⇒ MIR lowering of stacks of left joins",
-        default: false,
+        default: true,
```

Comment:
@ggevay / @mgree

Reply:
I would leave this off for self-hosted! There are various issues, e.g., https://github.com/MaterializeInc/database-issues/issues/8396

Also, this has the potential for a substantial improvement only with `enable_eager_delta_joins`, but that's not turned on, because that also has various issues.